Fix: Update button colors on Models pages

AresPhoenix345 opened the pull request:
**Description**
This PR fixes #2389 by updating the button colors on the Models pages to match the design requirements.

**Changes Made**
- Changed primary action buttons (`.import`) to saffron yellow (#EBC017)
- Changed secondary action buttons (`.import-secondary`) to keppel green (#00B39F)
- Added tertiary button class (`.import-tertiary`) with transparent background and 1px solid border
- Updated "Add New Model" button to use secondary styling
- Updated "Quick Start" button in modal to use secondary styling

**Files Modified**
- `_sass/catalog.scss` - Updated button color classes
- `_layouts/single-page-model.html` - Changed "Add New Model" button class
- `_includes/models/modal.html` - Changed "Quick Start" button class

**Testing**
- ✅ Verified button colors on model pages
- ✅ Verified button colors in modal
- ✅ Tested responsive design on multiple screen sizes

**Screenshots**
<img width="1920" height="1080" alt="Screenshot 2025-10-21 225358" src="https://github.com/user-attachments/assets/5518369e-4695-48f1-9457-05718fadf5f5" />
<img width="1920" height="1080" alt="Screenshot 2025-10-21 225504" src="https://github.com/user-attachments/assets/4adba47b-701f-4d98-8758-32d421310c1a" />

**Notes for Reviewers**
Please verify the color values match the design specifications in #2389.

**[[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

saurabhraghuvanshii commented on _sass/catalog.scss:
```diff
@@ -204,8 +204,16 @@ p {
+}
+.import-tertiary {
+  background-color: transparent;  // TERTIARY: Transparent background
+  border: 1px solid #00b39f;
```
@AresPhoenix345  where you're using this?

saurabhraghuvanshii commented on _sass/catalog.scss:
```diff
@@ -189,7 +189,7 @@ p {
 
 .import {
-  background-color: #00b39fff;
+  background-color: #EBC017;  // PRIMARY: Saffron yellow
```
@Rajesh-Nagarajan-11 Is saffron our primary color? I’m not sure.

saurabhraghuvanshii replied:
@AresPhoenix345 I need confirmation from @Rajesh-Nagarajan-11 first. After that, you can address the other changes.

kishore08-07 reviewed (changes requested):
@AresPhoenix345
Re-read the issue description and the desired state.The issue requires three action states (Primary, Secondary, and Tertiary), but only two are currently implemented.Add the tertiary action as specified.

AresPhoenix345 commented:
> @AresPhoenix345 Re-read the issue description and the desired state.The issue requires three action states (Primary, Secondary, and Tertiary), but only two are currently implemented.Add the tertiary action as specified.

@kishore08-07 Referring to the diff, the tertiary-action exists, I am not sure what the comment is about? please clarify.

kishore08-07 commented:
> > @AresPhoenix345 Re-read the issue description and the desired state.The issue requires three action states (Primary, Secondary, and Tertiary), but only two are currently implemented.Add the tertiary action as specified.
> 
> @kishore08-07 Referring to the diff, the tertiary-action exists, I am not sure what the comment is about? please clarify.

@AresPhoenix345 
I can’t find the tertiary button action being implemented here.
<img width="1017" height="625" alt="image" src="https://github.com/user-attachments/assets/6759af73-d9dd-47cb-9a5c-e13f2caa8e2c" />

AresPhoenix345 committed:
fix: Update button colors on Models pages

- Changed primary button to saffron yellow (#E0C077)
- Changed secondary button to keppel green (#00b39f)
- Added tertiary button with transparent background

Signed-off-by: Aditya Tiwari <phoenixcodeforces@gmail.com>

saurabhraghuvanshii committed:
Merge branch 'master' into fix/buttonsOnModelsPage

AresPhoenix345 committed:
Change import background color to a darker shade

Signed-off-by: Awanish Tripathi <phoenixcodeforces@gmail.com>

AresPhoenix345 commented:
> @AresPhoenix345 saffron is not saffron Please check

@saurabhraghuvanshii It was mentioned to be saffron yellow, however I have made it darker in shade.
Do let me know if it is to be just saffron.

saurabhraghuvanshii commented:
@AresPhoenix345 look lgtm!!

saurabhraghuvanshii commented:
@AresPhoenix345 yeah lgtm!!, but I would suggest you to present this item in upcoming meeting so that other can also share their thoughts because it's ui change. Or discuss on slack once.

Rajesh-Nagarajan-11 reviewed (changes requested):
Can you implement a hover effect for these buttons?

AresPhoenix345 committed:
Merge branch 'master' into fix/buttonsOnModelsPage

AresPhoenix345 commented:
Hi @kishore08-07 I was a bit occupied in academics, will update this soon to add hover effects as instructed by @Rajesh-Nagarajan-11.

saurabhraghuvanshii committed:
Merge branch 'master' into fix/buttonsOnModelsPage

AresPhoenix345 committed:
Merge branch 'master' into fix/buttonsOnModelsPage

AresPhoenix345 committed:
Add hover effect to the modes

Signed-off-by: Aman Yadav <phoenixcodeforces@gmail.com>

rishiraj38 commented:
Stale PR.
